### DISCUSSION TITLE
Config xml (doublecmd.xml)

### DISCUSSION THREAD

**Discussion by vertigo220** [doublecmd/doublecmd#1980] [Suggestions]
I find configuring settings in this file to be unnecessarily difficult. Clicking the button to open the file opens it in Double Commander, which is not ideal since any changes made will be reverted since DC is running. It would make much more sense to open it in a different program, not to mention I prefer using my standard text editor, as I'm more familiar with it and it has much more functionality. There should at least be an option, if not made default, to open the xml file in the default text editor. Currently, I have to manually locate the file and open it in my own editor.

I also think it would be helpful to make this have it's own button at the bottom, between the "Help" and "OK" buttons, so it's more visible. This would make it far more obvious there are additional settings as well as make it easier to access, which I believe makes sense considering how many of DC's settings are handled by the file.

Alternately, or in addition to that, a couple other options would be to have the search display options located in the config file, and clicking them would open the file for editing (again, preferably in the default editor), or to display the file's contents (or just all the different options it contains) on their own tab in options where they can be edited and applied.

**Comment by Skif-off** [comment]
[Edit & Apply](https://doublecmd.github.io/doc/en/configuration.html#ConfigDC)?
